packages/tupaia-web/src/layout/ProjectCardList/ProjectCard.tsx

```diff
@@ -0,0 +1,175 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+import React, { ComponentType } from 'react';
+import styled from 'styled-components';
+import Lock from '@material-ui/icons/Lock';
+import Alarm from '@material-ui/icons/Alarm';
+import { darken } from '@material-ui/core/styles';
+import { Typography } from '@material-ui/core';
+import { SingleProject } from '../../types';
+import { Link } from 'react-router-dom';
+import { Button } from '@tupaia/ui-components';
+import { USER_ROUTES } from '../../Routes';
+
+const Card = styled.div`
+  display: flex;
+  flex-direction: column;
+  padding: 1.6rem;
+  min-height: 20rem;
+  border-radius: 5px;
+  background: ${({ theme }) => theme.projectCard.background};
+  color: white;
+  box-sizing: border-box;
+  align-items: flex-start;
+  justify-content: space-between;
+
+  @media (min-width: ${({ theme }) => theme.breakpoints.values.sm}px) {
+    padding: 2.5rem;
+    height: 24rem;
+  }
+
+  button {
+    margin-top: auto;
+  }
+`;
+
+const Logo = styled.div`
+  position: relative;
+  background: white;
+  width: 5rem;
+  height: 5rem;
+  border-radius: 3px;
+  overflow: hidden;
+  margin-bottom: 0.625rem;
+
+  > img {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    max-width: 100%;
+    max-height: 100%;
+  }
+`;
+
+const Title = styled(Typography)`
+  font-size: 1.25rem;
+  line-height: 1.4;
+  font-weight: 500;
+  margin-bottom: 0.625rem;
+`;
+
+const Text = styled(Typography)`
+  font-size: 0.875rem;
+  line-height: 1.2;
+  margin-bottom: 0.625rem;
+`;
+
+const CountryText = styled(Text)`
+  color: ${({ theme }) => theme.projectCard.subText};
```

Comment:
Do you think it would be appropriate to set the default material ui theme.palette.text.secondary color to this color?

Reply:
I think it would have knock on effects - everywhere else the secondary text is white, but this isn't

Reply:
My understanding is that the theme primary text color is currently the default for a dark the which is white and the secondary is the default which is: color: rgba(255,255,255,0.7);

I can't see any where that we are using the secondary text color yet. Can you?